GVFS/GVFS/CommandLine/DiagnoseVerb.cs

```diff
@@ -50,9 +50,10 @@ protected override void Execute(GVFSEnlistment enlistment)
                 this.WriteMessage(string.Empty);
                 this.WriteMessage("gvfs version " + ProcessHelper.GetCurrentProcessVersion());
 
-                GitVersion gitVersion;
-                string error;
-                if (GitProcess.TryGetVersion(out gitVersion, out error))
+                GitVersion gitVersion = null;
+                string error = null;
+                string gitPath = GVFSPlatform.Instance.GitInstallation.GetInstalledGitBinPath();
+                if (!string.IsNullOrEmpty(gitPath) && GitProcess.TryGetVersion(gitPath, out gitVersion, out error))

```

Comment:
Use `out string _` like you did in GVFSVerb?

Reply:
we are writing the error here to console, in case TryGetVersion fails.